packages/keycloak-service/src/service/serviceClass.ts

```diff
@@ -95,7 +99,7 @@ type paramsType = URLParams | null;
  * **To update an object**: service.update(theObject)
  */
 export class KeycloakAPIService {
-  public accessToken: string;
+  public accessTokenOrCallBack: typeof handleSessionOrTokenExpiry;
```

Comment:
shouldn't this too allow a string type i.e : `string | () => Promise<string>`

Reply:
> @KipSigei, question here, why did we write a keycloack-service as opposed to re-using the opensrp-service

Yes we did, this is just incase we end up having different request methods/headers for keycloak